packages/wonder-blocks-link/src/__tests__/__snapshots__/custom-snapshot.test.tsx.snap

```diff
@@ -114,13 +114,17 @@ exports[`LinkCore kind:primary href:# light:false visitable:false focused 1`] =
   onTouchStart={[Function]}
   style={
     {
+      ":focusVisible": {
+        "borderRadius": 3,
+        "color": "#1865f2",
+        "outline": "1px solid #1865f2",
+      },
       "alignItems": "center",
-      "borderRadius": 3,
       "color": "#1865f2",
       "cursor": "pointer",
       "display": "inline-flex",
-      "outline": "1px solid #1865f2",
-      "textDecoration": "none",
+      "outline": "none",
+      "textDecoration": "underline currentcolor solid",
```

Comment:
**question:** why does `textDecoration` now changes to this value?

Reply:
@jandrade I believe that comes from the `restingInline` style, now that I removed the conditional before resting styles.